apps/search-widget-demo/src/App.svelte

```diff
@@ -125,7 +125,7 @@
         lang="en"
         placeholder="Search"
         {prompt}
-        filters="labels,created"
+        preselected_filters="/classification.labels/artist/Mademoiselle K"
```

Comment:
Maybe better to do it with a variable like `prompt` so it is easy to comment/comment out.